openequella/openEQUELLA
# Selenium tests for Duplicate Check

* Test Editbox duplicate check
* Test attachments(files and links) duplicate check
* Test Duplicate data page
* Add a new collection to institution fiveo
* Make attachment warning consistent with Editbox warning
